test/tests/executioners/eigen_convergence/b.i

```diff
@@ -0,0 +1,73 @@
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = 0
+    xmax = 160
+    ymin = 0
+    ymax = 160
+    nx = 8
+    ny = 8
+  []
+  uniform_refine = 0
+[]
+
+[Variables]
+  [u]
+  []
+[]
+
+[Kernels]
+  [diffusion]
+    type = MatDiffusion
+    diffusivity = diffusivity
+    variable = u
+  []
+  [reaction]
+    type = CoefReaction
+    coefficient = 0.01
+    variable = u
+  []
+  [rhs]
+    type = MassEigenKernel
+    coefficient = 0.01
+    variable = u
+  []
+[]
+
+[BCs]
+  [robin]
+    type = VacuumBC
+    boundary = 'left bottom'
+    variable = u
+  []
+[]
+
+[Materials]
+  [nm]
+    type = GenericConstantMaterial
+    block = 0
+    prop_names = 'diffusivity'
+    prop_values = 0.333333333333333333
+  []
+[]
+
+[Postprocessors]
+  [fluxintegral]
+    type = ElementIntegralVariablePostprocessor
+    variable = u
+    execute_on = linear
+  []
+[]
+
+[Executioner]
+  type = NonlinearEigen
+  bx_norm = fluxintegral
+  solve_type = PJFNK
+  free_power_iterations = 4
```

Comment:
Need to add `output_after_power_iterations = false` so that the tests can pass.

Reply:
are you talking about the sweep failures on next? Isn't this just about adjusting the tolerance?

Reply:
For the failures. This will disable the output after free  power iterations which can be affected by the number of processors.

Reply:
It can be fixed by setting l_tol possibly so that each power iteration we get a fixed behavior regardlless of number of processors, but we can just turn this output off.

Reply:
Oh I understand now. I didn't catch that we were diffing on the "intermediate" result

Reply:
Then I can possibly remove the `rel_err` setting in the `tests` spec as well 👍